### PR TITLE
completed css-layout-classes

### PR DESCRIPTION
**Pull request by Ajowyang**
Screenshot 2024-07-17 at 4 57 57 PM
Screenshot 2024-07-17 at 4 58 02 PM

### CODE REVIEW

**Comment by Ajowyang** [comment]
Are my Kirby fox pikachu and ness larger than they should be?

File: css-layout-classes/styles.css [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  /* max-width: 800px; */
+  display: flex;
+  justify-content: center;
+}
+
+.columnQuarter {
+  display: flex;
+  max-width: 200px;
+  justify-content: center;
+}
```
Comment (brandon-moy) [inline css-layout-classes/styles.css]: While technically yes, 200px is 1/4 of your 800px, what happens if you change your `container`'s `max-width`? Then you have to come back down and adjust any rules that are also modified by it. Instead you should be using a percentage.

File: css-layout-classes/index.html [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+            owned by Nintendo and created by Japanese video game designer
+            Shigeru Miyamoto. Acting as the company's mascot, as well as being
+            the eponymous protagonist of the series, Mario has appeared in over
+            200 video games since his creation.
+          </p>
+          <p>
+            Depicted as a short, pudgy, Italian plumber who resides in the
+            Mushroom Kingdom, his advanetures generally center upon rescuing
+            Princess Peach from the Koopa villain Bowser.
+          </p>
```
Comment (brandon-moy) [inline css-layout-classes/index.html]: Should these be `column-half`? In the reference image are they `half` or more/less than `half` for each piece?

Also I don't see a CSS ruleset for `columnHalf` so where was this getting styled from?

**Review by brandon-moy — CHANGES_REQUESTED** [review]
Great start but a couple of things here as well as in comments:

Your four images in a row are larger than intended to be. This is because while you have your container items for them set to a specific width, you are not specifying the width of the image element to max that.

Also when you name classes in HTML/CSS, you should use `kebab-case` rather than `camelCase`. `camelCase` is used more in your TypeScript for variables and functions.

**Review by brandon-moy — APPROVED** [review]
👍

**Pull request merged**